extra/smart-proxy-openscap-send.cron

```diff
@@ -1,2 +1,2 @@
 # Send all collected OpenSCAP reports once every 30 minutes
-*/30 * * * * foreman-proxy smart-proxy-openscap-send >>/var/log/foreman-proxy/cron.log 2>&1
+*/30 * * * * foreman-proxy smart-proxy-openscap-send >>/var/log/foreman-proxy/cron.log > /dev/null
```

Comment:
last comment - I think you can remove `> /dev/null`, we already redirect STDOUT to cron.log